var/spack/repos/builtin/packages/tensorflow/package.py

```diff
@@ -0,0 +1,344 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+from glob import glob
+
+
+class Tensorflow(Package):
```

Comment:
Should we rename the package to `py-tensorflow`?

Reply:
That's fine with me.

Reply:
It isn't, but that doesn't mean we can't rename it. See https://spack.readthedocs.io/en/latest/build_systems/pythonpackage.html#choosing-a-package-name for the docs I wrote on how I normally decide on `py-` vs. no `py-`. I think I'm leaning towards `py-`? TF has bindings for other languages, but I think it's primarily used via Python. I'm not sure it's even possible to disable the Python bindings.